release_notes.md

```diff
@@ -0,0 +1,58 @@
+# What's New
+
+- This solution pack is now **Certified**
+- **Summary** tab contains more details and now has following sections available as part of this solution pack's details:
+    - A new category **Compliance and Reporting** &ndash; helps with filtering from the left pane when using the check-boxes under **Category**
+    - A new tag **GDPR** &ndash; helps with filtering from the left pane when using the **Search By Tags** option
+    - Added instructions to configure "Code Snippet" connector before execution of GDPR playbooks
+- A new module **Data Compliance** has been intoduced
+- A new Navigation menu added for 'Data Compliance'
+- Added new playbooks to **Check for SLA violation** and to **Send Reminder for SLA Breach**
+- Updated **GDPR Assessment Report** to be compitable with newly created module i.e 'Data Compliance'
+- Added a provision to execute GDPR playbooks either in **Test Mode** or in **Production Mode**
+- The task and te form associated with **Notify Affected Individuals** now also shows list of email-IDs of the Affected Individuals
```

Comment:
te form?

Reply:
It should be "the form", Updated the same in release_notes.md.